assets/src/components/map.tsx

```diff
@@ -238,7 +238,11 @@ const Map = (props: Props): ReactElement<HTMLDivElement> => {
         appState.pickerContainerIsVisible
       )
     }
-  }, [shouldAutoCenter, latLngs, appState.pickerContainerIsVisible])
+  }, [
+    shouldAutoCenter,
+    JSON.stringify(latLngs),
```

Comment:
So how does this end up fixing the bug? Is it possible to write a test case to prevent a regression? Or maybe just a description here if not—I can imagine this line looking strange/unnecessary out of context.

Reply:
Yeah, it is pretty strange. Added a comment.